### PR TITLE
Update windows-prerequisites.md

### PR DESCRIPTION
**Pull request by jchannon**
## Summary
Updates windows pre-requisuites after finding an issue with compiled stand alone .net core applications

See here https://github.com/dotnet/cli/issues/5201 https://github.com/dotnet/cli/issues/195

## Details
When trying to run a stand alone application I received an error as shown in https://github.com/dotnet/cli/issues/5201 A comment was made in https://github.com/dotnet/cli/issues/195 that says Visual C++ is a dependency so this PR elaborates on existing docs for the scenario where you have a stand alone app.

### CODE REVIEW

File: docs/core/windows-prerequisites.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ You can see the full set of [supported operating systems](https://github.com/dot
 
 ## .NET Core dependencies
 
-.NET Core requires the VC++ Redistributable when running on Windows. It is installed for you by the .NET Core installer. You need to install the Visual C++ redistributable manually if you are installing .NET Core via the installer script (`dotnet-install.ps1`). 
+.NET Core requires the VC++ Redistributable when running on Windows. It is installed for you by the .NET Core installer. You need to install the Visual C++ redistributable manually if you are installing .NET Core via the installer script (`dotnet-install.ps1`) or are attempting to run a compiled stand alone .NET Core application.
```
Comment (svick) [inline docs/core/windows-prerequisites.md]: [This article](https://docs.microsoft.com/en-us/dotnet/articles/core/deploying/#self-contained-deployments-scd) calls it "self-contained", not "stand alone". I think the terminology should be consistent.

Also, not sure what "compiled" is supposed to mean here, all self-contained apps are compiled.

So, I would change this to:

> … or are attempting to run a self-contained .NET Core application.

**Comment by jchannon** [comment]
@svick updated. Also added another commit after discussing the docs with @gkhanna79  on https://github.com/dotnet/cli/issues/195

File: docs/core/windows-prerequisites.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ ms.devlang: dotnet
 ms.assetid: c33b1241-ab66-4583-9eba-52cf51146f5a
 ---
 
-# Prerequisites for Windows development
+# Prerequisites for Windows development and/or running .Net Core applications
```
Comment (mairaw) [inline docs/core/windows-prerequisites.md]: .NET needs to be all caps

How about we change the title to something like "Prerequisites for .NET Core on Windows"?

File: docs/core/windows-prerequisites.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ ms.devlang: dotnet
...
 
 .NET Core development on Windows with Visual Studio requires:
```
Comment (mairaw) [inline docs/core/windows-prerequisites.md]: I'd probably change a few sections around. For example, I'd create a section for Visual Studio after the general information about .NET Core on Windows and move this info further down. But I can make the changes for you after we merge this.

File: docs/core/windows-prerequisites.md [hunk 4]
```diff
@@ -36,16 +36,13 @@ You can see the full set of [supported operating systems](https://github.com/dot
...
-.NET Core requires the VC++ Redistributable when running on Windows. It is installed for you by the .NET Core installer. You need to install the Visual C++ redistributable manually if you are installing .NET Core via the installer script (`dotnet-install.ps1`). 
+.NET Core requires the VC++ Redistributable when running on Windows. It is installed for you by the .NET Core installer. You need to install the Visual C++ redistributable manually if you are installing .NET Core via the installer script (`dotnet-install.ps1`) or are attempting to run a self-contained .NET Core application. 
 
-The Visual C++ Redistributable version differs by Windows version.
+[Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145)
 
-* Windows 10
-    * [Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145)
-* Windows 7+ (not Windows 10)
+* Windows 7 & 2008 only
```
Comment (mairaw) [inline docs/core/windows-prerequisites.md]: Windows 7 & 2008 -> Windows 7 and Windows 2008

**Review by mairaw — COMMENTED** [review]
Thanks @jchannon! I've left a few comments. @blackdwarf @gkhanna79 do you guys want to review this?

**Comment by jchannon** [comment]
@mairaw updated 😄

**Review by gkhanna79 — APPROVED** [review]

**Review by mairaw — APPROVED** [review]
Looks good @jchannon! I'll wait for the CI builds to finish. Thanks for your contribution!

**Comment by jchannon** [comment]
Thanks. How soon until it appears on the live docs site?

**Comment by mairaw** [comment]
We need to merge the changes to the live branch. We just did one a few hours ago. We're close to get a few more PRs merged and then we can publish live again. So probably later today.

**Comment by BillWagner** [comment]
I was planning to run the script to update live later this evening (Eastern time)

**Pull request merged**